tests/e2e/helpers/accessibility-utilities.js

```
@@ -0,0 +1,1001 @@
+          results.cssFeatures[feature] = test();
+        } catch (e) {
+          results.cssFeatures[feature] = false;
+          results.warnings.push(`CSS feature test failed: ${feature}`);
+        }
+      });
+      
+      // Test JavaScript feature support
+      const jsTests = {
+        'ES6 Classes': () => !!window.class,
+        'Arrow Functions': () => {
+          try { eval('(() => {})'); return true; } catch(e) { return false; }
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)